git/gitignore.symlink

```diff
@@ -5,3 +5,6 @@
 db/*.sqlite3.backup
 vendor/bundle
 .fs.yml
+.byebug_history
+.vscode
+.solargraph.yml
```

Comment:
We often add to this file:

coverage
.bundle
node_modules

Also Sublime project can have names, so it can be added like

*.sublime-project
*.sublime-workspace

Reply:
@vladimirbazhanov added in the https://github.com/fs/dotfiles/pull/36/commits/04c26a87e1c0f6ce0542db78df14b306d00a41fe